File: sol/solution/8 kyu/5a0be7ea8ba914fc9c00006b/Solution.jl
#=
Solution
=#
function sakura_fall(v)
    if v <= 0
        return 0.0
    else
        return 400.0 / v
    end
end





#=
Sample Tests
=#
using FactCheck

facts("sakura_fall") do

  context("basic tests") do
    @fact sakura_fall(5) --> roughly(80; atol=1e-6)
    @fact sakura_fall(10) --> roughly(40; atol=1e-6)
    @fact sakura_fall(200) --> roughly(2; atol=1e-6)
    @fact sakura_fall(-1) --> roughly(0; atol=1e-6)
  end
end

File: sol/solution/8 kyu/5a0be7ea8ba914fc9c00006b/Solution.cs
/*
Solution
*/
public static class Kata
{
    public static double SakuraFall(double v)
    {
        if (v <= 0) return 0.0;
        return 400.0 / v;
    }
}





/*
Sample Tests
*/
using NUnit.Framework;

[TestFixture]
public class SampleTests
{
    [Test]
    public void SampleTest()
    {
        Assert.That(
            Kata.SakuraFall(5),
            Is.EqualTo(80).Within(1e-6));
    }
}